SpywareFilter/sections/general_url.txt

```diff
@@ -58,6 +58,9 @@
 /plausible.js
 ||plausible.io/js/p.js
 ||plausible.io/api/event
+! Privacy Counter
+||privacycounter.com/js/script.js
+||privacycounter.com/api/event
```

Comment:
1. Remove your rules.
2. Add `||privacycounter.com^$third-party` to the `tracking_servers.txt`.